docs/system-overview/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+permalink: /system-overview/
+
+# Banner Content
+# =================================
+title: System Overview
+subtitle:
+hero_image: assets/img/system-diagrams/system-diagram.svg
+---
+
+<div class="system-overview-page" markdown="1">
+
+## Single Computer Setup
+
+![Single Computer Diagram](/digital_slide_archive/assets/img/system-diagrams/system-diagram-single-computer-setup.svg "Single Computer Diagram"){:width="75%"}
```

Comment:
If you make these relative links, they will work in the artifacts, too (e.g., instead of `/digital_slide_archive/assets/img/system-diagrams/system-diagram-single-computer-setup.svg`, use `../assets/img/system-diagrams/system-diagram-single-computer-setup.svg`).